qiita_pet/templates/analysis_results.html

```diff
@@ -1,8 +1,15 @@
 {% extends sitebase.html%}
 
 {%block head%}
-
-
+<link rel="stylesheet" href="/static/vendor/css/select2.min.css" type="text/css">
+<script type="text/javascript" src="/static/vendor/js/select2.min.js"></script>
+<script type="text/javascript" src="/static/js/sharing.js"></script>
+<script type="text/javascript">
+$(document).ready(function () {
+  current_id = {{analysis_id}};
+  update_share('analysis');
```

Comment:
Exactly. There is no way of passing in current_id there as a parameter if it is not a global variable since it is unknown, so modifying the function will not work. I have a way of using data variables I am testing right now.